waspc/src/Wasp/AI/GenerateNewProject/InitialFiles.hs

```diff
@@ -21,6 +21,7 @@ import Wasp.AI.GenerateNewProject.Common
     getProjectPrimaryColor,
   )
 import Wasp.AI.GenerateNewProject.Plan (PlanRule)
+import Wasp.Generator.WebAppGenerator.Common (reactVersion)
```

Comment:
Ha, at first I wanted to comment that this is a bit wild, that Wasp/AI code imports react version from the Generator, but then I realized it actually makes sense.
Hm I am not sure, part of me wants to not have this dependency, to keep AI somewhat independent of Generator, but yeah this does keep us in sync automatically, so it might be fine.

What about other deps here, just below: typescript, prisma -> you are not pulling those versions from Generator. Following this way of reasoning, we should also import those? I guess what I would say: let's import all of let's import none. I am ok wtih importing none also. Or at least have TODO toimport those although that is a bit lazy.

Reply:
I've went with _importing none_ since for all other templates we are not dynamically injecting the versions but rather we'll validate them when this is done: #2317